apps/admin/frontend/src/api.ts

```diff
@@ -382,3 +392,17 @@ export const addPrintedBallots = {
     });
   },
 } as const;
+
+export const setSystemSettings = {
+  useMutation() {
+    const apiClient = useApiClient();
+    const queryClient = useQueryClient();
+    return useMutation(apiClient.setSystemSettings, {
+      async onSuccess() {
+        await queryClient.invalidateQueries(
+          getCurrentElectionMetadata.queryKey()
```

Comment:
I think `getCurrentElectionMetadata` should be `getSystemSettings` since that's the query we want to invalidate (probably just a copy-paste error)

Reply:
Thanks for the catch, indeed a copy and paste error https://github.com/votingworks/vxsuite/pull/3147/commits/d6b5efd7f572c85c2a475a53edd0db0c76de2cc6#diff-a21e7f57fc6e4d06d99aa89c6921cbc36789c5bd69e48f5c617f3805eb0c9943R424